daemon/src/model.rs

```diff
@@ -792,4 +793,42 @@ mod tests {
             )],
         );
     }
+
+    #[test]
+    fn initial_funding_fee() {
+        // Initial funding fee is taken for the whole settlement period, in this
+        // case 1%
+
+        let funding_params = FundingParams::new(
+            Amount::from_sat(100),
+            FundingRate::from(dec!(0.01)),
+            SETTLEMENT_INTERVAL.whole_hours(),
+        )
+        .unwrap();
+
+        assert_eq!(funding_params.calculate_fee().unwrap(), 1);
+    }
+
+    use proptest::prelude::*;
+
+    proptest! {
+        #[test]
+        fn rollover_funding_fee_throughout_a_day_should_amount_at_least_to_daily_founding_fee(amount_sat in 1_000u64..1_000_000u64) {
+            let funding_params_daily =
+                FundingParams::new(Amount::from_sat(amount_sat), FundingRate::from(dec!(0.01)), SETTLEMENT_INTERVAL.whole_hours()).unwrap();
```

Comment:
Nit: The test assumes that `SETTLEMENT_INTERVAL == 24`. That might not always be true and I can guarantee you we will forget updating the name of this test once we change that constant. I'd inline that value here because the test is actually unrelated to the constant.

Reply:
fair enough! I certainly can see how we'd forget to change it :) renamed the test name, variables and assert message.